### PR TITLE
add IDE format configuration files from previous projects

### PR DESCRIPTION
**Pull request by slu-it**
I added the formatter configuration used in the previous version of LivingDoc and WebTester. There is also an ADR which will be accepted if this is merged.

### CODE REVIEW

File: dev/intellij_formatter.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<code_scheme name="testIt - Formatter">
```
Comment (benhamidene) [inline dev/intellij_formatter.xml]: we do not need testIT anymore .. why not LivingDoc

Comment (slu-it) [inline dev/intellij_formatter.xml]: good catch, i simply copied the files.. I'll change that asap

**Comment by slu-it** [comment]
I've changed the Formatter name for Eclipse and IntelliJ to `LivingDoc Formatter`.

**Comment by bitterblue** [comment]
Thanks! I'm not sure that we need an ADR for this. A common coding style throughout the code base is definitely good practice. I like to see ADRs used to explain hard design decisions and their trade-offs, not as an instrument for governance.

**Comment by Drakojin** [comment]
I agree.
IF we really want to include an ADR for this we have to at least give a more understandable reason than "we already used it".

**Comment by slu-it** [comment]
OK, I've removed the ADR

**Pull request merged**